azure-pipelines.yml

```diff
@@ -26,26 +26,19 @@ pool:
   vmImage: 'windows-latest'
 
 variables:
-  - group: nuget
   - name: testFolderPath
     value: '$(Build.SourcesDirectory)/test'
   - name: DOTNET_SKIP_FIRST_TIME_EXPERIENCE
-    value: 'true'
+    value: true
   - name: DOTNET_CLI_TELEMETRY_OPTOUT
-    value: 'true'
+    value: true
   - name: buildConfiguration
     value: Release
 
 steps:
 
   # Configure NuGet
-- pwsh: |
-    @('develop','release','AzurePipelines') | ForEach-Object{
-        dotnet nuget update  source $_ -p $env:NUGET_KEY -u VssSessionToken --configfile nuget.config
-    }
-  condition: and(succeeded(), variables['nuget.key'])
-  displayName: 'Add api token to access nuget artifacts'
-  env:
-    NUGET_KEY: $(nuget.key)
-
-- template: crawler.build.yml@templates
+  - task: NuGetAuthenticate@0
```

Comment:
@romaklimenko not sure about changes. In SqlServer hovewer we have this version.

Reply:
It this is the same as in SQL, then we are fine.